tracing/logging.go

```diff
@@ -11,7 +11,7 @@ const (
 	logKey = contextKey("nf-log-key")
 )
 
-func requestLogger(r *http.Request, log logrus.FieldLogger) logrus.FieldLogger {
+func requestLogger(r *http.Request, log logrus.FieldLogger) (logrus.FieldLogger, string) {
```

Comment:
this change is mostly to not call the `RequestID` multiple times. Only used in 1 place (and not exported) so I tossed it in